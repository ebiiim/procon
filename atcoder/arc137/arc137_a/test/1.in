151515 1000000000000000000
