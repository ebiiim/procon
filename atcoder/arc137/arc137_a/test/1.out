999999999999848483
